custom_dc/custom_data.md

```diff
@@ -46,17 +45,27 @@ San Francisco,2023,300,300,200,50
 San Jose,2023,400,400,300,0  
 ```
 
-The _ENTITY_ is an existing property in the Data Commons knowledge graph that is used to describe an entity, most commonly a place. The best way to think of the entity type is as a key that could be used to join to other data sets. The column heading can be expressed as any existing place-related property; see [Place types](../place_types.md) for a full list. You can also use `dcid` or its prefixes, `geoId`, `latLng`. If you use `dcId`, the value must be a resolved [DCID](../glossary.md#dcid). If you're not sure what to use, you can simply use the heading `name` or `place` and the importer will resolve it automatically.
+The _ENTITY_ is an existing property in the Data Commons knowledge graph that is used to describe an entity, most commonly a place. The best way to think of the entity type is as a key that could be used to join to other data sets. The column heading can be expressed as any existing place-related property; see [Place types](../place_types.md) for a full list. It may also be any of the special DCID prefixes listed in (Special place names)[#special-names].
 
-The _DATE_ is the date of the observation and should be in the format _YYYY_, _YYYY_-_MM_, or _YYYY_-_MM_-_DD_, with the corresponding heading, i.e. `year`, `month` or `date`.
+The _DATE_ is the date of the observation and should be in the format _YYYY_, _YYYY_-_MM_, or _YYYY_-_MM_-_DD_. The heading can be anything, although as a best practice, we recommend using a corresponding identifier, such as `year`, `month` or `date`.
 
 The _VARIABLE_ should contain a metric [observation](../glossary.md#observation) at a particular time. We recommend that you try to reuse existing statistical variables where feasible; use the main Data Commons [Statistical Variable Explorer](https://datacommons.org/tools/statvar) to find them. If there is no existing statistical variable you can use, name the heading with an illustrative name and the importer will create a new variable for you. 
 
 The variable values must be numeric. Zeros and null values are accepted: zeros will be recorded and null values ignored.
 
 All headers must be in camelCase.
 
-The following are some valid examples of headers:  
+### Special place names {#special-names}
+
+In addition to the place names listed in [Place types](../place_types.md), you can also use (`dcid`)[../glossary.md#dcid] or the following special prefixes:
```

Comment:
Consider adding small blurbs for each. You can use the below for content:

#### dcid
An already resolved DC ID.

e.g. `country/USA`, `geoId/06`, etc.

#### country3AlphaCode
3 character country codes.

e.g. `USA`, `CHN`, etc.

#### geoId
Place geo IDs.

e.g. `06`, `023`, etc.

#### wikidataId
Wikidata place identifiers.

e.g. `Q12345`

#### lat#lng
Latitude and longitude of the place using the format `<lat>#<lng>`.

e.g.  `38.7#-119.4`

Reply:
Thanks for this info!  Added.  So is lat#lng not using camel case? I.e. it's not lat#Lng?

Reply:
I think it's case insensitive but we can use `lat#lng` in this case for documentation.